06-11/DorothyGrahamBirthday.markdown

```diff
@@ -0,0 +1,3 @@
+06.11.1944 Dorothy Graham was born in the US.She  has been in software testing for 40 years, and is co-author of 4 books: Software Inspection, Software Test Automation, Foundations of Software Testing and Experiences of Test Automation.
```

Comment:
Dates in text description seem inconsistent. We should use a MM/DD as the directory structure for easier machine indexing. OTOH for the text we should use DD Mon Year everywhere. This one seems not to follow this. 

Note: this should be noted in readme under a contributing/structure section or something like that.